identity-service/src/notifications/formatNotificationMetadata.js

```diff
@@ -413,7 +452,7 @@ const pushNotificationMessagesMap = {
     return `${capitalize(notification.sendingUser.name)} became your #${notification.rank} Top Supporter!`
   },
   [notificationTypes.SupportingRankUp] (notification) {
-    return `You're now ${notification.receivingUser.name}'s #${notification.rank} Top Supporter!`
+    return `You're now ${capitalize(notification.receivingUser.name)}'s #${notification.rank} Top Supporter!`
```

Comment:
Don't think we want to capitalize this here - was capitalizing the others since it's the start of the sentence